calculate_average_ddimtirov.sh

```diff
@@ -17,5 +17,7 @@
 
 # --enable-preview to use the new memory mapped segments
 # We don't allocate much, so just give it 1G heap and turn off GC; the AlwaysPreTouch was suggested by the ergonomics
-JAVA_OPTS="--enable-preview -Xms1g -Xmx1g -XX:+UnlockExperimentalVMOptions -XX:+UseEpsilonGC  -XX:+AlwaysPreTouch"
+JAVA_OPTS="--enable-preview -da -dsa -Xms1g -Xmx1g -XX:+UnlockExperimentalVMOptions -XX:+UseEpsilonGC  -XX:+AlwaysPreTouch"
```

Comment:
Please add ` -XX:-EnableJVMCI` t omit jvm warning 
```
[0.001s][warning][gc,jvmci] Setting EnableJVMCI to false as selected GC does not support JVMCI: epsilon gc
```

Reply:
This is likely because the wrong VM has been used (I assume Temurin, not GraalVM).
My new PR should fix that.